apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.html.ts

```diff
@@ -0,0 +1,46 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+export function AIGeneratePage({ resLocals }: { resLocals: Record<string, any> }) {
```

Comment:
```suggestion
export function AiGeneratePage({ resLocals }: { resLocals: Record<string, any> }) {
```

Reply:
Done in ff0b8fae364583747e838607ffc674c57e963e85.